lightning/src/ln/channelmanager.rs

```diff
@@ -73,12 +73,14 @@ use core::ops::Deref;
 use std::time::Instant;
 
 mod inbound_payment {
+	use alloc::string::ToString;
```

Comment:
Not needed?

Reply:
`cargo test --verbose --color always --no-default-features --features no-std` (part of CI) fails without that line

Reply:
Ah, I see. Seems other uses are covered by `use prelude::*;`

Reply:
I don't get a compile warn here.

Reply:
https://github.com/lightningdevkit/rust-lightning/actions/runs/1590076364

Reply:
Oh, no, I meant I don't get a compile warn that this use is unnecessary.

Reply:
Ah, just haven't seen that `use` elsewhere in the repo when needing `to_string`